### DISCUSSION TITLE
How to begin with it?

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [vfsfitvnm/frida-il2cpp-bridge#79] [Q&A]
I wanted to get the dump.cs of my android game. I've root and pc too.
Can I please know how to get started with it?

**Comment by vfsfitvnm** [comment]
Hi, I apologize but a decent frida/programming knowledge is required

**Comment by RevealedSoulEven** [comment]
Actually I use frida inject as I've a rooted android device but I've no idea how to write that java script for this🥲 can you please help with a java code example for this?

**Reply by JulianWinter1124** [reply]
Try to follow the guide here: [Installation](https://github.com/vfsfitvnm/frida-il2cpp-bridge/wiki/Installation). I had to fiddle a bit with the project json files a bit but otherwise this got me started.

> Actually I use frida inject as I've a rooted android device but I've no idea how to write that java script for this🥲 can you please help with a java code example for this?
